lib/db/memory.js

```diff
@@ -57,6 +60,16 @@ const unique = require('../unique');
  *       userId: <user_id>,
  *       type: <string>,
  *       scope: <string>,
+ *       createdAt: <timestamp>,
+ *       expiresAt: <timestamp>
+ *     }
+ *   },
+ *   refreshTokens: {
+ *     <token>: {
+ *       token: <string>,
+ *       clientId: <client_id>,
+ *       userId: <user_id>,
+ *       scope: <string>,
  *       createdAt: <timestamp>
```

Comment:
Do you think it would be useful to track a `lastUsedAt` for refresh tokens?  We might want to e.g. expire tokens that have not been used in months.  Doesn't have to be part of the initial landing, but something to think about.